src/QirTools/pyqir/qirlib/src/emit/intrinsics.rs

```diff
@@ -0,0 +1,95 @@
+use inkwell::module::Module;
+use inkwell::values::FunctionValue;
+
+pub const M: &str = "M";
```

Comment:
How will this work for instruction sets other than those represented in Microsoft.Quantum.Intrinsic? Will it be possible for users to emit (for example) calls to MS or IsingZZ gates on platforms that support those instructions?